.github/workflows/scripts/linux/appimage.sh

```diff
@@ -49,12 +53,12 @@ cp --dereference /lib/"$LIBARCH"/libgcc_s.so.1 "$GITHUB_WORKSPACE"/squashfs-root
 chmod +x .github/workflows/scripts/linux/app-variables.sh
 cp .github/workflows/scripts/linux/app-variables.sh "$GITHUB_WORKSPACE"/squashfs-root/apprun-hooks
 export UPD_INFO="gh-releases-zsync|PCSX2|pcsx2|latest|$name.AppImage.zsync"
-/tmp/squashfs-root/AppRun --appdir="$GITHUB_WORKSPACE"/squashfs-root/ --plugin gtk -d "$GITHUB_WORKSPACE"/squashfs-root/PCSX2.desktop -i "$GITHUB_WORKSPACE"/squashfs-root/PCSX2.png
+/tmp/squashfs-root/AppRun --appdir="$GITHUB_WORKSPACE"/squashfs-root/ --plugin qt -d "$GITHUB_WORKSPACE"/squashfs-root/PCSX2.desktop -i "$GITHUB_WORKSPACE"/squashfs-root/PCSX2.png
 # see LD_LIBRARY_PATH in app-variables.sh - the intent is to use system wayland if available but fall back to app-image provided
 # a little bit hacky but should ensure maximum compatibility
-mkdir -p squashfs-root/usr/lib/wayland
-mv squashfs-root/usr/lib/libwayland-* squashfs-root/usr/lib/wayland
-rm squashfs-root/usr/lib/libgmodule-2.0.so.0
+#mkdir -p squashfs-root/usr/lib/wayland
```

Comment:
Comments?

Reply:
left it in as out-of-scope for qt, but still part of wX

Reply:
This app image is for Qt, not WX